charts/etos/charts/suite-starter/templates/deployment.yaml

```diff
@@ -36,3 +44,17 @@ spec:
           envFrom:
           - configMapRef:
               name: etos
+        {{- if .Values.global.filebeatSidecar.enabled }}
```

Comment:
Indentation.

Reply:
Indentation is correct in the generated yaml. If I indent, I cannot deploy.